### DISCUSSION TITLE
How to create chapter list ?

### DISCUSSION THREAD

**Discussion by kayungjo** [TortugaPower/BookPlayer#713] [Q&A]
How to create chapter list ?
The audio books I downloaded comes with bunch of .mp3 and .marker files.
.marker file show the chapters of the book and indicate time when each chapter starts.
(ex. <Marker><Name>Book 1 - Chapter 3</Name><Time>20:14.000</Time></Marker>)
Is there a way I can add chapter list to my playlist or to added audio files
instead of using bookmark ?

**Comment by GianniCarlo** [comment]
We have this right now in the open beta on TestFlight (link to join: https://testflight.apple.com/join/FLv3zLyN) we'll be releasing it though for everybody by the end of the week if you'd like to wait
